bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/ReolinkHandler.java

```diff
@@ -322,6 +322,11 @@ public void channelRead(@Nullable ChannelHandlerContext ctx, @Nullable Object ms
                         ipCameraHandler.setChannelState(CHANNEL_ENABLE_RECORDINGS, OnOffType.ON);
                     }
                     break;
+                case "/api.cgi?cmd=Reboot":
+                    if (!content.contains("\"rspCode\" : 200")) {
+                        ipCameraHandler.logger.warn("Reboot failed:\n{}", content);
+                    }
+                    break;
```

Comment:
This would be a good place to set the channelstate to off?

Reply:
Agree, it needs to reflect the state of the device. I believe this is one reason why a thing action is normally used for reboot commands.
https://www.openhab.org/docs/developer/bindings/#actions-bound-to-a-thing

Channels are used when the device has a state that changes ie ON and OFF... So the channel should turn OFF if it fails and also turn OFF after it has rebooted. Otherwise this should be implemented as an ACTION where you do not track the state.

Reply:
Ok, the other comments are fixed. I’ll wait for you to implement the action before we continue